resources/zib/zib-ApgarScore.10Minute.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-ApgarScore.10Minute" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-ApgarScore.10Minute" />
```

Comment:
Based on the profiling guidelines this profile should be called _zib-ApgarScore-10Minute_ (the same holds for the other profiles). See section 12.1.1.1 of the profiling guidelines: _If there are multiple implementations of the same building block for different uses, a use specific suffix is appended._

Reply:
Done